frontend/model/state.js

```diff
@@ -109,7 +126,8 @@ const mutations = {
     } else {
       Vue.set(state.currentChatRoomIDs, state.currentGroupId, null)
     }
-  }
+  },
+  updateVuexExtensionState () {}
```

Comment:
Technically it's a no-op mutation. So maybe it could be called just `noop()`, provided the explanative comment later at line 523 is given. Maybe a short comment here would be good too.

Reply:
I like that idea / name of `noop`, great suggestion 👍